rosi.py

```diff
@@ -0,0 +1,12 @@
+from discord.ext import commands
+
+class Rosi(commands.Bot):
+    def __init__(self):
+        super().__init__(command_prefix='!', description="MCSS Discord Bot")
+        self.__config = {}
+
+    def set_config(self, config):
```

Comment:
Nothing should really be setting the config after boot up so we should kill this method.